korgan/miniapp_api_v4.py

```
@@ -0,0 +1,582 @@
+    x_telegram_init_data: str = Header(default=""),
+) -> dict[str, Any]:
+    identity = core.legacy._identity(x_telegram_init_data)
+    await core.legacy._require_consent(identity)
+    _require_admin(identity)
+    order = await get_document_order(order_id)
+    if order is None:
+        raise HTTPException(status_code=404, detail="Платёжный запрос не найден")
+    if order.status != "awaiting_admin":
+        raise HTTPException(status_code=409, detail="Этот платёжный запрос уже обработан")
+    if not await decide_document_order(order.id, approved=payload.approved, note=payload.note):
+        raise HTTPException(status_code=409, detail="Не удалось зафиксировать решение: статус уже изменился")
```

Ruff (0.16.2)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)